packages/@aws-cdk/aws-servicecatalogappregistry/README.md

```diff
@@ -99,6 +99,45 @@ const associatedApp = new appreg.ApplicationAssociator(app, 'AssociatedApplicati
 });
 ```
 
+If you want to associate an Attribute Group with application created by `ApplicationAssociator`, then use as shown in the example below:
+
+```ts
+import * as cdk from "@aws-cdk/core";
+
+const app = new App();
+
+class CustomAppRegistryAttributeGroup extends cdk.Stack {
+  public readonly attributeGroup: appreg.AttributeGroup
+  constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+    const myAttributeGroup = new appreg.AttributeGroup(app, 'MyFirstAttributeGroup', {
+      attributeGroupName: 'MyAttributeGroupName',
+      description: 'Test attribute group',
+      attributes: {},
+    });
+
+    this.attributeGroup = myAttributeGroup;
+  }
+}
```

Comment:
Do we really expect people to define a stack for their AttributeGroup? Couldn't it go into one of the other stacks?

For example, wouldn't it make sense for a user to do:

```ts
const associatedApp = new appreg.ApplicationAssociator(app, 'AssociatedApplication', {
  applications: [ /* ... */],
  attributes: { 
    // ...
  },
});
```

Reply:
`attributes` is not a valid input as per [ApplicationAssociatorProps](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-servicecatalogappregistry/lib/application-associator.ts#L10). Attribute group can definitely be a part of some other stack, however customers need to explicitly associate the attribute group to the application, as `ApplicationAssociator` is responsible to associate the stacks within the scope, it wont automatically associate the underlying AGs within a stack.

Reply:
> attributes is not a valid input as per ApplicationAssociatorProps

I understand. 

The nice thing about software is that it is not like the laws of physics. It is in fact changeable. 

* FACT: "attributes" can not be passed to the ApplicationAssociator
* QUESTION: would it be better/easier/more convenient/more obvious IF "attributes" could be passed to ApplicationAssociator?

Reply:
TBH I like this idea of including `attribute-group` in `application-associator` construct. However, can we first prioritize this bug fix which will unblock our customers and they should be able to associate an attribute group to an application.

I will do some more deep dive on including `attribute-group` in `application-associator`, its pros and cons and will let you know about our plan to modify `application-associator` in order to include `attribute-group`.

Let me know if this plan works for you.